docs/ops/normalization/LRN_1.md

```diff
@@ -6,70 +6,74 @@
+Each input value is divided by
+\f[ (bias + \frac{alpha}{{size}^{len(axes)}} \cdot \sum_{i} data_{i})^{beta} \f]
+The sum is taken over a region of a side length `size` and number of dimensions equal to number of axes.
+The region is centered at the input value that's being normalized (with zero padding added if needed).
+
+Here is an example for 4D `data` input tensor and `axes = [1]`:
+```
+sqr_sum[a, b, c, d] =
+    sum(data[a, max(0, b - size / 2) : min(data.shape[1], b + size / 2 + 1), c, d] ** 2)
+output = data / (bias + (alpha / size ** len(axes)) * sqr_sum) ** beta
+```
+
+Example for 4D `data` input tensor and `axes = [2, 3]`:
+```
+sqr_sum[a, b, c, d] =
+    sum(data[a, b, max(0, c - size / 2) : min(data.shape[2], c + size / 2 + 1),  max(0, d - size / 2) : min(data.shape[3], d + size / 2 + 1)] ** 2)
+output = data / (bias + (alpha / size ** len(axes)) * sqr_sum) ** beta
+```
+
 **Attributes**:
 
 * *alpha*
 
-  * **Description**: *alpha* represents the scaling attribute for the normalizing sum. For example, *alpha* equal 0.0001 means that the normalizing sum is multiplied by 0.0001.
+  * **Description**: *alpha* represents the scaling attribute for the normalizing sum. For example, *alpha* equal `0.0001` means that the normalizing sum is multiplied by `0.0001`.
   * **Range of values**: no restrictions
-  * **Type**: float
+  * **Type**: `float`
   * **Default value**: None
   * **Required**: *yes*
 
 * *beta*
 
-  * **Description**: *beta* represents the exponent for the normalizing sum. For example, *beta* equal 0.75 means that the normalizing sum is raised to the power of 0.75.
+  * **Description**: *beta* represents the exponent for the normalizing sum. For example, *beta* equal `0.75` means that the normalizing sum is raised to the power of `0.75`.
   * **Range of values**: positive number
-  * **Type**: float
+  * **Type**: `float`
   * **Default value**: None
   * **Required**: *yes*
 
 * *bias*
 
   * **Description**: *bias* represents the offset. Usually positive number to avoid dividing by zero.
   * **Range of values**: no restrictions
-  * **Type**: float
+  * **Type**: `float`
   * **Default value**: None
   * **Required**: *yes*
 
 * *size*
 
   * **Description**: *size* represents the side length of the region to be used for the normalization sum. The region can have one or more dimensions depending on the second input axes indices.
   * **Range of values**: positive integer
-  * **Type**: int
+  * **Type**: `int`
   * **Default value**: None
   * **Required**: *yes*
 
 **Inputs**
 
-* **1**: `data` - input tensor of any floating point type and arbitrary shape. Required.
+* **1**: `data` - tensor of type `T` and arbitrary shape. **Required.**
 
-* **2**: `axes` - specifies indices of dimensions in `data` that define normalization slices. Required.
+* **2**: `axes` - 1D tensor of type `T_IND` which specifies indices of dimensions in `data` that define normalization slices. **Required.**
```

Comment:
`that` -> `which`

Reply:
Changed in this [commit](https://github.com/openvinotoolkit/openvino/pull/5890/commits/81196d7d7cea3cf05b66013d434fd44e4e6b04d5).